iroh/src/rpc_protocol.rs

```diff
@@ -139,6 +118,38 @@ impl ServerStreamingMsg<ProviderService> for BlobDownloadRequest {
 #[derive(Debug, Clone, Serialize, Deserialize, derive_more::From, derive_more::Into)]
 pub struct BlobDownloadResponse(pub DownloadProgress);
 
+/// A request to the node to download and share the data specified by the hash.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct BlobExportRequest {
+    /// The hash of the blob to export.
+    pub hash: Hash,
+    /// Set to true if the `hash` referes to a collection and all children of the collection should
```

Comment:
Not a fan of "recursive" here. hash seqs are just 1 level deep, so if somebody is going to put HashSeqs in HashSeqs this won't do what they expect. Not sure what the best name is though.

Reply:
Make it another enum, with values Blob and Collection? (this won't work for arbitrary HashSeqs)

Reply:
Sounds good. Added an enum.